spec/requests/api/services_spec.rb

```diff
@@ -354,4 +354,162 @@ def expect_svc_with_vms
       expect_bad_request("Cannot expand subcollection vms by name and virtual attribute")
     end
   end
+
+  describe "Power Operations" do
+    describe "start" do
+      it "will start a service for a user with appropriate role" do
+        service = FactoryGirl.create(:service)
+        api_basic_authorize(action_identifier(:services, :start))
+
+        run_post(services_url(service.id), :action => "start")
+
+        expected = {
+          "href"    => a_string_matching(services_url(service.id)),
+          "success" => true,
+          "message" => a_string_matching("starting")
+        }
+        expect(response.parsed_body).to include(expected)
+        expect(response).to have_http_status(:ok)
+      end
+
+      it "can start multiple services for a user with appropriate role" do
+        service_1, service_2 = FactoryGirl.create_list(:service, 2)
+        api_basic_authorize(collection_action_identifier(:services, :start))
+
+        run_post(services_url, :action => "start", :resources => [{:id => service_1.id}, {:id => service_2.id}])
+
+        expected = {
+          "results" => a_collection_containing_exactly(
+            a_hash_including("success"   => true,
+                             "message"   => a_string_matching("starting"),
+                             "task_id"   => anything,
+                             "task_href" => anything,
+                             "href"      => a_string_matching(services_url(service_1.id))),
+            a_hash_including("success"   => true,
+                             "message"   => a_string_matching("starting"),
+                             "task_id"   => anything,
+                             "task_href" => anything,
+                             "href"      => a_string_matching(services_url(service_2.id))),
+          )
+        }
+        expect(response.parsed_body).to include(expected)
+        expect(response).to have_http_status(:ok)
+      end
+
+      it "what happens when it fails to start"
```

Comment:
@syncrou is there an easy way to create a service that will fail to start? Or @abellotti should I be looking for a failure to queue the start?